### DISCUSSION TITLE
Reversing a subtitle

### DISCUSSION THREAD

**Discussion by thepwrtank18** [SubtitleEdit/subtitleedit#6536] [Q&A]
I downloaded an SSA, which looks like this:
![image](https://user-images.githubusercontent.com/57580668/208328267-8ff1acc6-47bf-4e40-8679-6a131f1706ca.png)

See how the start time goes down as the list goes on. The beginning's at the bottom. How do I reverse this so that the beginning is at the top, like normal? This way, I can convert it to an SRT properly.

**Comment by GrampaWildWilly** [comment]
On Sun, Dec 18, 2022, 19:43 TheTank20 ***@***.***> wrote:

Copy/paste the complete subtitle file to Notepad++.  Use the function for
sorting all lines in reverse lexicological order.  Presto magic.

**Comment by darnn** [comment]
There is a thing in SE itself that does this: Tools->Sort by->Start time.
